runtime-builder/Dockerfile

```diff
@@ -12,13 +12,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Dockerfile that runs a App Engine flexible environment Go application.
+# Dockerfile for building a Go application and generating the corresponding
+# application Dockerfile for it.  It expects the following build-args:
+#   go_version - Go SDK version to be bundled in this image.
+#   debian_tag - DEBIAN tag for use when generating application Dockerfile.
+# When this builder image is run, it expects the application source files to be
```

Comment:
I actually did mean to say "run" as "running" is progressive tense.  I changed it to "executed" instead, perhaps that may be clearer.